pkg/iut/v1alpha1/v1alpha1.go

```diff
@@ -134,13 +137,12 @@ func (h V1Alpha1Handler) Start(w http.ResponseWriter, r *http.Request, ps httpro
 
 	var startReq StartRequest
 	if err := json.NewDecoder(r.Body).Decode(&startReq); err != nil {
-		responses.RespondWithError(w, http.StatusBadRequest, err.Error())
+		RespondWithError(w, http.StatusBadRequest, err.Error())
 		return
 	}
-	defer r.Body.Close()
```

Comment:
The body is being read here and needs to be closed. The difference between this one and `Status` is that `Status` does not do `json.NewDecoder(r.Body).Decode(&...)`